airbyte-integrations/bases/base-normalization/main_dev_transform_catalog.py

```diff
@@ -3,7 +3,16 @@
 #
 
 
+import logging
+
+from airbyte_cdk.exception_handler import init_uncaught_exception_handler
+from airbyte_cdk.utils.traced_exception import AirbyteTracedException
 from normalization.transform_catalog.transform import main
 
 if __name__ == "__main__":
-    main()
+    init_uncaught_exception_handler(logging.getLogger("airbyte"))
+    try:
+        main()
+    except Exception as e:
+        msg = "Something went wrong while transforming the catalog in Normalization. See the logs for more details."
+        raise AirbyteTracedException(str(e), msg, exception=e)
```

Comment:
Will this message be shown to users (AirbyteTracedException), perhaps:

> Something went wrong while normalizing the data moved in this sync. See the logs for more details.

Reply:
how much detail do we want in those user-facing messages? I think it's good to state that the error happened in catalog transformation, i.e. we didn't even try to touch the "big data" at all:

> Something went wrong while normalizing the data moved in this sync (failed to transform catalog into dbt project). See the logs for more details.

(admittedly: this is a bit bikesheddy; ideally this never happens because we have really solid test coverage on the two `transform_*.py` files, and they don't interact with the destination warehouse/db at all)